docs/tutorials/mongoid-upgrade.txt

```diff
@@ -23,6 +23,48 @@ Upgrading to Mongoid 7.3
 
 The following sections describe significant changes in Mongoid 7.3.
 
+
+``delete`` Method Does Not Trigger Association Dependent Behavior
+----------------------------------------------------------------------
+
+**Breaking change:** In Mongoid 7.3,
+:ref:`dependent behavior <dependent-behavior>` is not invoked
+when the parent association is deleted using the ``delete`` method.
+For example, after the following code snippet executes, in Mongoid 7.3 the
+album will remain in the database:
+
+.. code-block:: ruby
+
+  class Band
+    include Mongoid::Document
+
+    has_many :albums, dependent: :destroy
+  end
+  
+  class Album
+    include Mongoid::Document
+
+    belongs_to :band
+  end
+  
+  band = Band.create!
+  album = Album.create!(band: band)
+  
+  # Does not delete the album from the database
+  band.delete
+
+Previous versions of Mongoid invoked dependent behavior when deleting parents.
+
+To invoke dependent behavior, use the ``destroy`` method:
+
+.. code-block:: ruby
+  
+  # Deletes the album from the database
+  band.destroy
+
+The behavior of Mongoid 7.3 is consistent with how ActiveRecord behaves.
```

Comment:
> The behavior of Mongoid 7.3 is consistent with how ActiveRecord behaves.

The behavior of Mongoid 7.3 is consistent with how [ActiveRecord associations](https://guides.rubyonrails.org/association_basics.html) behave.

Reply:
I prefer to not link to AR docs when not necessary to avoid link rot as well as sending users away from our documentation.